### PR TITLE
add rnn usage doc

### PR DESCRIPTION
**Pull request by phlrain**
add rnn usage

### CODE REVIEW

**Comment by phlrain** [comment]
image

File: doc/fluid/api/api_guides/low_level/layers/index.rst [hunk 1]
```diff
@@ -14,4 +14,5 @@
     loss_function.rst
     data_in_out.rst
     control_flow.rst
+    rnn_usage.rst
```
Comment (tink2123) [inline doc/fluid/api/api_guides/low_level/layers/index.rst:17]: 为避免多人同时操作index文件产生冲突，请不要在PR中提交index文件的修改~

**Comment by shanyi15** [comment]
Hi, we've read this guide and here is the feedback:
- Lack of some important content, please take [sequence api guide](http://www.paddlepaddle.org/documentation/docs/zh/1.1/api/api_guides/low_level/layers/sequence.html) as an example
- Please pay attention to the grammar and punctuation, making the sentences more fluent

please fix these ASAP, thanks very much

**Comment by shanyi15** [comment]
由于超过6个月没有更新，此PR先行关闭
please feel free to reopen it, thx

**Pull request closed without merging**